### PR TITLE
Adding a swift compatible set representation options method to the fi…

### PR DESCRIPTION
**Pull request by jdizoglio**
…le request class

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @jdizoglio has signed the CLA. Thanks for the pull request!

File: BoxContentSDK/BoxContentSDK/Requests/BOXFileRequest.m [hunk 1]
```diff
@@ -262,6 +262,23 @@ - (void)setRepresentationRequestOptions:(BOXRepresentationRequestOptions)represe
...
 
+// Swift compatable va_list version of the method
+- (void)setRepresentationRequestOptions:(BOXRepresentationRequestOptions)representationOptions args:(va_list)argumentList
+{
+    NSMutableArray *arguments=[[NSMutableArray alloc]init];
+    BOXRepresentationRequestOptions eachObject;
+    if (representationOptions) {
+        [arguments addObject: [NSNumber numberWithUnsignedInteger:representationOptions]];
+        while ((eachObject = va_arg(argumentList, BOXRepresentationRequestOptions))) {
+            if ([NSNumber numberWithUnsignedInteger:eachObject] > 0) {
```
Comment (jlawton) [inline BoxContentSDK/BoxContentSDK/Requests/BOXFileRequest.m]: This will always be `> 0`, because it's non-`nil`

Comment (jdizoglio) [reply BoxContentSDK/BoxContentSDK/Requests/BOXFileRequest.m]: done

**Comment by jdizoglio** [comment]
retry

**Comment by jdizoglio** [comment]
test this please

**Review by jlawton — APPROVED** [review]

**Pull request merged**